test/conftest.py

```diff
@@ -28,6 +33,15 @@ def pytest_collection_modifyitems(items):
         # 'needs_cuda' mark, and the ones with device == 'cpu' won't have the
         # mark.
         needs_cuda = item.get_closest_marker("needs_cuda") is not None
+        needs_ffmpeg_cli = item.get_closest_marker("needs_ffmpeg_cli") is not None
+        has_skip_marker = item.get_closest_marker("skip") is not None
+        has_skipif_marker = item.get_closest_marker("skipif") is not None
```

Comment:
QQ @mollyxu , if we use 

```py
@pytest.mark.skipif(cond, ...)
```

and `cond` is `False`, does the test still have the mark? I just want to make sure we're not skipping tests that shouldn't be skipped. It might be worth adding a small comment :) !